datasets/urdu_sentiment_corpus/README.md

```diff
@@ -0,0 +1,138 @@
+---
+annotations_creators:
+  - expert-generated
+language_creators:
+  - crowdsourced
+languages:
+  - ur
+licenses: []
```

Comment:
is there no licenses or is it unknown ?
if unknown please use 
```
licenses:
- unknown
```

Reply:
Yeah sure... will make the change.